code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm

```diff
@@ -240,6 +240,8 @@
 				suit = pickweight(list(/obj/item/clothing/suit/hooded/explorer = 18, /obj/item/clothing/suit/hooded/cloak/goliath = 2))
```

Comment:
The suit is better than the goliath cloak because it can be reinforced, please alter this for balance. (I don't like either cloak or suit being here but having to choose 1 of 2 to remove the suit has to go)